algo HW1.2.py

```diff
@@ -0,0 +1,20 @@
+'''
+Count odd and even numbers. Count odd and even digits of the whole number.
+Example: entered number is 34560, then 3 digits will be even (4, 6, and 0) and 2 odd digits (3 and 5).
+'''
+
+
+def original_number(i):
```

Comment:
The time complexity for this code will be O(n2).
It's possible to make it better. Good job anyway!